### DISCUSSION TITLE
Snipe IT Granular Permissions Help/Future Request

### DISCUSSION THREAD

**Discussion by brianhoganm** [snipe/snipe-it#11619] [Q&A]
Hello,

I would like to give permissions for a specific users to view assets in their location So, in other words I would like to give a user view only access to view  assets in their location only. I don't want them to see all the assets at the "company" level. This is great for teachers if they want to see all of the laptops checked out to their students. So, basically, I would put all of the laptops or devices in a location and whoever is assigned to that location can view all of the assets. They would be able to view who the devices are checked out. 

I also read this on Snipe IT documentation -  https://snipe-it.readme.io/docs/user-permissions and it appears to me if I understand it correctly, that you can create _new permissions_ for handling granular permissions used throughout Snipe-IT. 

**Has anyone tried to follow these directions?** 
Can anyone provide a little more instruction on how to do this? I'm not scared of editing code but I am just not sure how to approach this or if I am even understanding the article. 

**UPDATE**:  @snipe 
I tried following the directions https://snipe-it.readme.io/docs/user-permissions by using their example Florb and I am stuck. I'm not sure what I am doing wrong. I copied their example to see what would happen but I didn't see any changes in the UI.

Thanks,
Brian

**Comment by bilal-wistech** [comment]
@snipe I tried following the directions [snipe-it.readme.io/docs/user-permissions](https://snipe-it.readme.io/docs/user-permissions) by using their example Florb and I am stuck. I'm not sure what I am doing wrong. I copied their example to see what would happen but I didn't see any changes in the UI.
Same thing. I am also having issue in this. How to solve this issue?
